shepherd

```diff
@@ -9,21 +9,24 @@ update_services() {
   local blacklist="$1"
   local supports_detach_option=$2
   local supports_registry_auth=$3
+  local supports_insecure_registry=$4
   local detach_option=""
   local registry_auth=""
+  local insecure_registry_flag=""
   local name
   local apprise_sidecar_url="${APPRISE_SIDECAR_URL:-}"
 
   [ $supports_detach_option = true ] && detach_option="--detach=false"
   [ $supports_registry_auth = true ] && registry_auth="--with-registry-auth"
+  [ "$supports_insecure_registry" = true ] && insecure_registry_flag="--insecure"
```

Comment:
To be consistent, please remove the quotes around the first variable. (Also, I am not sure it works otherwise?)

Reply:
I have added this quote after a check with [ShellCheck](https://github.com/koalaman/shellcheck/wiki/SC2086).
As you ask, I have removed theses to be consistent.

Reply:
Ah, right. We should probably change the whole script to adhere to ShellCheck at one point.